daemon/oci_linux.go

```diff
@@ -73,7 +73,7 @@ func withLibnetwork(daemon *Daemon, daemonCfg *dconfig.Config, c *container.Cont
 					s.Hooks = &specs.Hooks{}
 				}
 				shortNetCtlrID := stringid.TruncateID(daemon.netController.ID())
-				s.Hooks.Prestart = append(s.Hooks.Prestart, specs.Hook{
+				s.Hooks.Prestart = append(s.Hooks.Prestart, specs.Hook{ //nolint:staticcheck // FIXME(thaJeztah); replace prestart hook with a non-deprecated one.
 					Path: filepath.Join("/proc", strconv.Itoa(os.Getpid()), "exe"),
```

Comment:
Used `//nolint` comments for these, and not changed these to `CreateRuntime` hooks for now after some discussion with @robmry and @akerouanton 

(the prestart hook won't be _removed_ from the runtime spec in the v1.x.x releases, so they remain functional for now)

Reply:
Yes, we're planning to remove `withLibnetwork` (again) soon. The Nvidia one will need another look at some point though.